packages/map-template/src/components/MapsIndoorsMap/MapsIndoorsMap.jsx

```diff
@@ -70,11 +70,18 @@ function MapsIndoorsMap({ apiKey, gmApiKey, mapboxAccessToken, venue, locationId
     /**
      * When venue is fitted while initializing the data, set map to be ready.
      */
-    function venueChangedOnMap() {
+    function venueChangedOnMap(venue) {
         if (isMapReady === false) {
             setMapReady(true);
         }
-        getVenueCategories(currentVenueName);
+
+        // If a venue is passed from the Map component, use that to get the venue categories,
+        // otherwise use the venue that has been set in the prop to get the categories.
+        if (venue) {
```

Comment:
Are we ever in a situation where this is not passed?

Reply:
Hmm, the venue is always passed. Would it be more correct to check if a `currentVenueName` is set, and if yes, use that, otherwise always default to the `venue` that is passed?

Reply:
No, we always pass the `venue`. Should we remove the check at all in this case and always assign the `venue.name`?

Reply:
✅